debian/control

```diff
@@ -991,6 +996,8 @@ Package: python3-cephfs
 Architecture: linux-any
 Section: python
 Depends: libcephfs2 (= ${binary:Version}),
+         python3-rados (= ${binary:Version}),
+         python-ceph-argparse (= ${binary:Version}),
```

Comment:
does this mean python3-cephfs requires python2 on debian?

Reply:
@smithfarm thanks for reviewing this change. that's was an oversight. 

fixed and repushed.